### PR TITLE
Add multicountdown app

### PR DESCRIPTION
**Pull request by BestDistress**
# Description
Added a mutlicountdown app.

# Copilot

### Summary
📄🐍⏲️

This pull request adds a new app called multicountdown, which allows the user to display the countdown to multiple events on their tidbyt device. The app consists of two files: `manifest.yaml`, which contains the app's metadata and configuration, and `multicountdown.star`, which contains the app's logic and user interface. The app uses Starlark, a dialect of Python, and some built-in modules for rendering, schema, and time.

> _`multicountdown` app_
> _Starlark code and manifest_
> _autumn events near_

### Walkthrough
*  Add multicountdown app that displays the current time and the countdown to the closest event among a series of events specified by the user ([link](https://github.com/tidbyt/community/pull/2026/files?diff=unified&w=0#diff-4196db674221d15d229260c6e3690bc33f0555df812550fc6525ce90dfa46789R1-R8), [link](https://github.com/tidbyt/community/pull/2026/files?diff=unified&w=0#diff-a23b99eeb8fbcbfe45317d5db9fbda5af1a7cf26f5bb25b64bcacfd4504c5975R1-R174))
   * Define main function that takes a config object and returns a render object with the app's user interface ([link](https://github.com/tidbyt/community/pull/2026/files?diff=unified&w=0#diff-a23b99eeb8fbcbfe45317d5db9fbda5af1a7cf26f5bb25b64bcacfd4504c5975R1-R174))
   * Define get_schema function that returns a schema object with the app's configuration fields and options ([link](https://github.com/tidbyt/community/pull/2026/files?diff=unified&w=0#diff-a23b99eeb8fbcbfe45317d5db9fbda5af1a7cf26f5bb25b64bcacfd4504c5975R1-R174))
   * Define helper functions for generating additional fields, finding the closest non-negative time difference, and formatting a duration ([link](https://github.com/tidbyt/community/pull/2026/files?diff=unified&w=0#diff-a23b99eeb8fbcbfe45317d5db9fbda5af1a7cf26f5bb25b64bcacfd4504c5975R1-R174))
   * Use render, schema, and time modules for creating the app's user interface, configuration, and time manipulation ([link](https://github.com/tidbyt/community/pull/2026/files?diff=unified&w=0#diff-a23b99eeb8fbcbfe45317d5db9fbda5af1a7cf26f5bb25b64bcacfd4504c5975R1-R174))
* Add `manifest.yaml` file with the app's metadata and configuration, such as its id, name, summary, description, author, file name, and package name ([link](https://github.com/tidbyt/community/pull/2026/files?diff=unified&w=0#diff-4196db674221d15d229260c6e3690bc33f0555df812550fc6525ce90dfa46789R1-R8))

### CODE REVIEW

**Comment by BestDistress** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by BestDistress** [comment]
recheck

File: apps/multicountdown/multicountdown.star [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                        ),
+                    ],
+                ),
+            ],
+        ),
+    )
+
+def more_options(eventCount):
+    returnArray = []
+    if eventCount > "0":
```
Comment (matslina) [inline apps/multicountdown/multicountdown.star:113]: Would you mind putting an upper limit on eventCount? Just someone entering a very large number doesn't cause weird failures that are confusing to users. I'm thinking 20 would be more than enough?

Comment (BestDistress) [reply apps/multicountdown/multicountdown.star:116]: No problem, done.

**Comment by BestDistress** [comment]
:shipit:

**Pull request merged**